examples/README.md

```diff
@@ -0,0 +1,109 @@
+in order to run successfully. The configuration files in in the `examples` directory 
+contain examples for the different product configurations. 
+
+This README provides a listing of all of the fields and descriptions.
+Fields that are optional are labeled accordingly.
+
+```
+version:
+    The version of the adapter implementation.
+
+billing_interval:
+    Sets the billing interval time period.
+
+api: (OPTIONAL)
+    Provides the path to the application information the csp-billing-adapter reports to the CSP API.
+    This is shared with the application the csp-billng-adapter is reporting about. The application may provide this as an endpoint directly, i.e. http://localhost/$API works.
+
+product_code:
+    TBD
+
+query_interval:
+    Time unit how often the csp-billng-adapter collectd the data from the application.
+
+reporting_api_is_cumulative:
+    Specifies if the CSP will accumulate reported values. While all CSPs will accumulate the values reported, it may not be possible to report 0. For example, in a monthly billing scenario, the csp-billing-adapter may have to report on a hourly basis a number that is not 0 such that the CSP API accumulates the values. A true setting would direct the adapater to normalize the unit count based on some to be determined normalization algorithm.
+
+reporting_interval: 
+    Sets the time when csp-billing-adapter reports to the CSP API. The values the csp-billing-adapter reports are determined by a combination of the settings for `reporting_api_is_cumulative` and `billing_interval`.
+
+support_info_collection_cmd:
+    A command executed by the adapter to collect specific information that will be useful for support situations.
+
+usage_metrics: 
+
+  {metric name}:
+    The name of the metric, for example managed_node_count
+
+    consumption_reporting:
+        Sets the way the csp-billing-adapter reports the value(s) it receives, either volume or tiered.
+
+        The volume value triggers the code to search for the highest possible dimension to use and all other configured dimensions will be reported as 0, or not reported, depending on CSP API requirements. For example:
+        
+            - The configuration is set to volume for the consumption_reporting
+            - The dimensions configuration contains 3 dimensions with a maximum setting of 30, 40, and 55
+            
+            If the value reported by the application API is 150:
+            
+                The csp-billing-adapter will report 0 for dimension 1 and 2 and report 150 for dimension 3.
+            
+            If the application API reports 36:
+
+                The csp-billing-adapter will report 0 for dimension 1, 36 for dimension 2, and 0 for dimension 3.
+            
+            In this case the application API may report only 1 dimension. If the application API reports more than 1 dimension it is an error condition.
```

Comment:
Having the error condition stated after the example may prompt the reader to tie the example together with the error condition statement as such it might help if we clarify as follows

"""If consumption_reporting is set to volume and the application API ...."